app/jobs/spotlight/rename_tags_job.rb

```diff
@@ -18,6 +18,7 @@ def perform(exhibit, tag, to: nil, **)
         all_tags += [to] unless to.nil?
 
         exhibit.tag(sidecar, with: all_tags, on: :tags)
+        sidecar.document.reindex
```

Comment:
Do we want to `document.reindex(update_params: {})` so it doesn't do the `commitWithin: 500` ?